index.html

```diff
@@ -26,12 +24,12 @@
         <section class="navigation_list">
             <nav class="navbar"> 
                 <!-- add space between items-->
-                <a class="navigation_item" href="#"> <strong>Meet Karma </strong> </a>
-                <a class="navigation_item" href="#"> How it Works </a>
-                <a class="navigation_item" href="#"> Store </a>
-                <a class="navigation_item" href="#"> Blog </a>
-                <a class="navigation_item" href="#"> Help </a>
-                <a class="navigation_item" href="#"> Login </a>
+                <a class="navigation_item" href="store.html"> <strong>Meet Karma </strong> </a>
+                <a class="navigation_item" href="store.html"> How it Works </a>
+                <a class="navigation_item" href="store.html"> Store </a>
+                <a class="navigation_item" href="store.html"> Blog </a>
+                <a class="navigation_item" href="store.html"> Help </a>
+                <a class="navigation_item" href="store.html"> Login </a>
```

Comment:
We need to link `Meet Karma` to homepage and `Store` to store page only. The rest should still be `#` because we don't have any pages for them yet.